### PR TITLE
Build options and descriptions

### PR DESCRIPTION
**Pull request by Jon-Salmon**
The project I'm working on requires a slightly different layout to the default one provided, notibly the exclution of the hotkey descriptions. I have hence added an options parameter to the `build` method with this functionality.

If you don't feel this will be useful to the project, feel free to ignore!

### CODE REVIEW

File: ShortcutCustomizeUI.js [hunk 1]
```diff
@@ -15,7 +15,11 @@ var ShortcutCustomizeUI = {
     return this.commonClass = `shortcut-customize-ui-${this.uniqueKey}`;
   },
 
-  build: async function() {
+  build: async function(options) {
+    var defaults = {
+      showDescription: true
```
Comment (piroor) [inline ShortcutCustomizeUI.js]: Could you change the name of the option to "showDescriptions" (plural form)?

**Comment by piroor** [comment]
Thanks! I think the option name should be "showDescriptions" (plural form), but almost OK.

**Comment by Jon-Salmon** [comment]
Thanks. Good spot, it makes more sense that way! I've updated it

**Comment by piroor** [comment]
Thanks!

**Pull request merged**